app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -401,14 +409,15 @@ class BrowserActivity : DuckDuckGoActivity(), BookmarkDialogCreationListener, We
     @SuppressLint("SetJavaScriptEnabled")
     private fun configureWebView() {
         webView = layoutInflater.inflate(R.layout.include_duckduckgo_browser_webview, webViewContainer, true).findViewById(R.id.browserWebView) as WebView
+        userAgentProvider = UserAgentProvider(webView.settings.userAgentString)
+
         webView.webViewClient = webViewClient
         webView.webChromeClient = webChromeClient
 
         webView.settings.apply {
+            userAgentString = userAgentProvider.getUserAgent()
             javaScriptEnabled = true
             domStorageEnabled = true
-            loadWithOverviewMode = true
-            useWideViewPort = true
```

Comment:
Why remove these two options from mobile? Overview mode was introduced to fix an issue https://github.com/duckduckgo/Android/pull/70.